geom/geom/src/TGeoManager.cxx

```diff
@@ -749,6 +749,27 @@ Int_t TGeoManager::AddRegion(TGeoRegion *region)
   return size;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Add a user-defined property
+
+void TGeoManager::AddProperty(TString property, Double_t value)
```

Comment:
Is the usage of TString in the interface is aligned with the rest of the interface? Maybe also we should avoid the copy.

Reply:
Consider using std::string_view for all string based interfaces (one 'exception' is the case where the the function will make a copy of the string anyway)

Reply:
Using now just const char * interfaces